CONTRIBUTING.md

```diff
@@ -318,10 +337,17 @@ These tests require several dependencies that can be installed either from the R
 | unidiff        | python3-unidiff (Zesty and higher) | unidiff        |
 | yamllint       | yamllint                          | yamllint       |
```

Comment:
This whole thing seems a bit redundant.  We tell the user 3 times to run `pytest` locally.

What I'm going to suggest instead is that we rewrite this to be simpler.  In particular, let's just add in one section called "Unit testing" or something like that.  In there, we say something like:

```
All of the dependencies for testing are listed in tests/requirements.txt.  It is recommended to use a virtual environment and pip to install the dependencies.
```

And then go on to give those instructions on how to install and run.  We can then remove the rest of the duplicated information here, so this is a lot shorter and more to the point.

Reply:
lgtm